patches/@shopify+flash-list+1.6.1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutView.kt b/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutView.kt
+index 6b78bd9..4571798 100644
+--- a/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutView.kt
++++ b/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutView.kt
+@@ -26,7 +26,7 @@ class AutoLayoutView(context: Context) : ReactViewGroup(context) {
+ 
+     /** Overriding draw instead of onLayout. RecyclerListView uses absolute positions for each and every item which means that changes in child layouts may not trigger onLayout on this container. The same layout
+      * can still cause views to overlap. Therefore, it makes sense to override draw to do correction. */
+-    override fun dispatchDraw(canvas: Canvas?) {
++    override fun dispatchDraw(canvas: Canvas) {
```

Comment:
Can you provide context of what issue this change fixes?

Reply:
It comes from `@Nullable` checks in newer Android versions which we are using right now and Kotlin is sensitive in that case.

Reply:
@WoLewicki upstream issue or PR for this change?

Reply:
Actually it is already released: https://github.com/Shopify/flash-list/pull/930 so we will do a folllow-up PR with bumping the version of the package after this PR is merged :tada: